tests/jsunit/gesture_test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Blockly Tests
+ *
+ * Copyright 2017 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /**
+ * @fileoverview Tests for gesture.
+ * @author marisaleung@google.com (Marisa Leung)
+ */
+'use strict';
+
+var e;
+var workspace;
+
+
+function gestureTest_setUp() {
+  workspace = new Blockly.Workspace();
+  e = {};
+}
+
+function gestureTest_tearDown() {
+  e = null;
+  workspace.dispose();
+}
+
+function test_gestureConstructor() {
+  var gesture = new Blockly.Gesture(e, workspace);
+  assertEquals(gesture.mostRecentEvent_, e);
+  assertEquals(gesture.creatorWorkspace_, workspace);
+}
+
+function test_gestureIsFieldClickInWorkspace() {
+  var block = new Blockly.Block(workspace);
```

Comment:
Ah I see what you are saying. When I ran the test by itself, it failed (as it should). I added the setUp and tearDown and it passed.

It must have been passing because of a left over workspace that hadn't been torn down correctly in another test.